django/cantusdb_project/main_app/forms.py

```diff
@@ -733,19 +714,9 @@ class Meta:
     #     help_text="RISM-style siglum + Shelf-mark (e.g. GB-Ob 202).",
     # )
 
-    shelfmark = forms.CharField(
-        required=True,
-        widget=TextInputWidget,
-    )
-
-    name = forms.CharField(
-        required=False,
-        widget=TextInputWidget
-    )
-
     holding_institution = forms.ModelChoiceField(
-        queryset=Institution.objects.all().order_by("name"),
-        required=True,
+        queryset=Institution.objects.all().order_by("siglum"),
```

Comment:
Is this still correct? As noted previously, this will cause Germany to fall under "D", Switzerland under "CH" and the UK under "GB".

Reply:
We can change to `city`. What I don't like about current is that it is hard to tell in the display what the order is. Siglum is marked off with parentheses and city is at the start of the display, so I think one of those is preferable to name.

![image](https://github.com/user-attachments/assets/3d158496-46be-46ff-bf38-f99e5b4c9bca)

Reply:
Whatever it is, it should be obvious in the display...

Reply:
Ok, I'll make it city then. I think alphabetical by first word the user sees is probably the most straightforward.